pkg/audit/containment.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+)
+
+// PendingAudit contains info needed for retrying an audit for a contained node
+type PendingAudit struct {
+	NodeID            storj.NodeID
+	PieceID           storj.PieceID
+	StripeIndex       int
+	ShareSize         int64
+	ExpectedShareHash []byte
+	ReverifyCount     int
+}
+
+// DB holds information about pending audits for contained nodes
+type DB interface {
+	Get(ctx context.Context, nodeID pb.NodeID) (*PendingAudit, error)
+	IncrementPending(ctx context.Context, pendingAudit *PendingAudit) error
+	Delete(ctx context.Context, nodeID pb.NodeID) error
+}
+
+// Containment allows access to pending audit info in the db
+type Containment struct {
```

Comment:
Where this abstraction is used or will be used?

Reply:
it will be used by the audit verifier. should I include that in the comment?

Reply:
I needed to ask because I see some code that it wasn't used, so if there wasn't plan to use it, I'd have ask for removing it, but it isn't the case.

> should I include that in the comment?

Maybe, it's a good idea and then we can remove the comment once it's used by the audit verifier.